jck/jtrunner/JavaTestRunner.java

```diff
@@ -237,29 +237,32 @@ public static boolean generateJTB() throws Exception {
 		// A known failures list (.kfl) file is optional.
 		// The automation here adds any files found (initial or updates) as 'custom' files. 
 		initialJtxFullPath = jckBase + "/lib/" + jckVersion + ".jtx";
+		File initialJtxFile = new File(initialJtxFullPath);
 
-		// Look for an update to the initial excludes file
-		if (jckVersion.contains("jck6") || jckVersion.contains("jck7")) {
-			jtxRelativePath = "excludes/jdk" + jckVersionNo + ".jtx";
-			kflRelativePath = "excludes/jdk" + jckVersionNo + ".kfl";
+		if (initialJtxFile.exists()) {
+			System.out.println("Using initial excludes list file " + initialJtxFullPath);
 		} else {
-			jtxRelativePath = "excludes/jck" + jckVersionNo + ".jtx";
-			kflRelativePath = "excludes/jck" + jckVersionNo + ".kfl";
+			System.out.println("Unable to find initial excludes list file " + initialJtxFullPath);
+			initialJtxFullPath = "";
 		}
-		
+
+		// Look for an update to the initial excludes file
+		jtxRelativePath = "excludes/jdk" + jckVersionNo + ".jtx";
```

Comment:
This should be "excludes/jck" not "excludes/jdk". Same change is needed for the kfl file too, in the next line. 

(The exclude files under `JCK11-unzipped/tree/master/excludes` are : jck11.jtx & jck11.kfl).

Reply:
An even simpler alternative for line 250 could be : `jtxRelativePath = "excludes/" + jckVersion + ".jtx"; `

Reply:
Thanks @Mesbah-Alam. Updated PTAL